test/WebSites/FormatterWebSite/Controllers/StreamController.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.AspNet.Mvc;
+using Microsoft.Net.Http.Headers;
+
+namespace FormatterWebSite
+{
+    public class StreamController : Controller
+    {
+        [HttpGet]
+        public Stream SimpleMemoryStream()
+        {
+            return CreateDefaultStream();
+        }
+
+        [HttpGet]
+        public Stream MemoryStreamWithContentType()
+        {
+            Response.ContentType = "text/html";
```

Comment:
Note that `FileStreamResult` does exactly that already, the name is there for back compat, but there is nothing File about `FileStreamResult`.